### ISSUE TITLE
Forward use:action to components

### ISSUE THREAD

**Issue by theetrain** [carbon-design-system/carbon-components-svelte#1599]
Let's add a way to pass actions to components, especially Form and FluidForm.

Proposed prop name is `use`. Example component:

```html
<script>
  export let use
</script>

<div use:use>
  Hello
</div>
```

Example usage:

```html
<script>
  import Component from './Component2.svelte'
	
  const action = () => console.log('action')
</script>

<Component use={action} />
```

Example: https://svelte.dev/repl/e24d07853ebe4773b1ab818ad60536a2?version=3.55.0

**Comment by theetrain** [comment]
Added proposal 2 above with a demo based on https://github.com/carbon-design-system/carbon-components-svelte/pull/1628#issuecomment-1383310189

Check out the demo; feedback is welcome.

**Comment by brunnerh** [comment]
> If defined, renders a `<form>` element, wrapping the target component

That does not seem right. If you want a form you can just add one; other components should not be concerned with this.
If someone actually needs the feature of single-button forms a lot, that can just be implemented in user land.

If a property for attributes is added, that should in my opinion refer to the element at hand and could just be called `attributes`.

So if you want a button and a form with a Svelte action:

```svelte
<Form action="?/kit-action" attributes={ useAction: enhance }>
    <Button type="submit">Submit</Button>
</Form>
```

**Comment by theetrain** [comment]
@brunnerh agreed; I reached a similar conclusion in my `<Button>` proposal in #1622. Single-element components like `<Button>` can have the user wrap the `<form>` themselves, or spread attributes such as `formaction` if they like.

For complex components with multiple buttons, such as the [sortable DataTable](https://carbon-components-svelte.onrender.com/components/DataTable#sortable), the user should be able to provide an `href` to pass in the same route with searchParams, or `formaction` and related attributes to each individual sortable button; allowing a user to sort a table without JS or hydration.
